src/main/java/org/isf/opd/gui/OpdBrowser.java

```diff
@@ -469,21 +567,29 @@ private JLabel getRowCounter() {
 	private Component getDateFilterPanel() {
 		if (dateFilterPanel == null) {
 			dateFilterPanel = new JPanel(new SpringLayout());
+			
+			dateFrom = new GoodDateChooser(LocalDate.now());
+			dateTo = new GoodDateChooser(LocalDate.now());
+			
+			resetDates();
 			dateFilterPanel.add(new JLabel(MessageBundle.getMessage("angal.common.datefrom.label")));
-			if (!GeneralData.ENHANCEDSEARCH) {
-				dateFrom = new GoodDateChooser(LocalDate.now().minusWeeks(1));
-			} else {
-				dateFrom = new GoodDateChooser(LocalDate.now());
-			}
 			dateFilterPanel.add(dateFrom);
 			dateFilterPanel.add(new JLabel(MessageBundle.getMessage("angal.common.dateto.label")));
-			dateTo = new GoodDateChooser(LocalDate.now());
 			dateFilterPanel.add(dateTo);
 			SpringUtilities.makeCompactGrid(dateFilterPanel, 2, 2, 5, 5, 5, 5);
 		}
 		return dateFilterPanel;
 	}
 
+	private void resetDates() {
+		if (!GeneralData.ENHANCEDSEARCH) {
+			dateFrom.setDate(LocalDate.now().minusWeeks(1));
```

Comment:
```suggestion
			dateFrom.setDate(TimeTools.getnow()minusWeeks(1));
```

Reply:
getNow() is returning a LocalDateTime, not a LocalDate, so it is not applicable in this case.
same for the GoodDateChooser constructor parameter

![image](https://user-images.githubusercontent.com/2938553/215572908-d0c50f71-85bc-49e1-858e-62ec2d64b7c4.png)

Reply:
you are right; I'm just moving too fast; ignore any others like that.

Reply:
ok, no prob! Thanks!

PS: we should think about something like getDateNow() and getDateTimeNow() ?